cmd/service.go

```diff
@@ -102,7 +102,11 @@ func (m *ServerMux) handleServiceSignals() error {
 			}
 			runExitFn(nil)
 		case serviceStop:
-			log.Println("Gracefully stopping... (press Ctrl+C again to force)")
+			log.Println("Exiting")
```

Comment:
Also a space before "Exiting" will be nice so that it prints
^C Exiting

and not

^CExiting